typing/src/main/java/org/navgurukul/typingguru/utils/TypingGuruPreferenceManager.kt

```diff
@@ -0,0 +1,31 @@
+package org.navgurukul.typingguru.utils
+
+import android.content.Context
+import android.content.SharedPreferences
+
+object TypingGuruPreferenceManager {
+    private const val PREF_NAME = "typing-guru"
+    private var mSharedPreferences: SharedPreferences? = null
+
+    fun init(context: Context) {
+        if (mSharedPreferences == null) {
+            mSharedPreferences = context.getSharedPreferences(
+                PREF_NAME,
+                Context.MODE_PRIVATE
+            )
+        }
+    }
+
+    private val editor: SharedPreferences.Editor
+        private get() = mSharedPreferences!!.edit()
+
+    fun setWebViewDisplayStatus(isShown: Boolean) {
+        val editor = editor
+        editor.putBoolean("isShown", isShown)
+        editor.commit()
```

Comment:
We can use a kotlin extension here which encapsulates boiler plate code. `SharedPreferences.edit{ }` can be used which takes a lambda as an argument where you can perform any edit operation.  Also we shouldn't use `commit()` and it is heavy on main thread. `SharedPreferences.edit{ }` used `apply` internally